### PR TITLE
Logic for automatic contract instances updates when Hub is updated

### PR DESCRIPTION
**Pull request by u-hubar**
This PR includes feature that on continuously running dkg.js process, if there happens update of ContentAsset contract, dkg.js will reinitialize contract instance with new contract address.

### CODE REVIEW

**Review by djordjekovac — APPROVED** [review]

File: .eslintrc.js [hunk 1]
```diff
@@ -6,7 +6,6 @@ module.exports = {
         node: true,
     },
     extends: ['airbnb/base', 'prettier'],
-    parser: 'babel-eslint',
```
Comment (djordjekovac) [inline .eslintrc.js:9]: Why did you remove linter chekc?

Comment (u-hubar) [reply .eslintrc.js:9]: I added it before to create a decorator, now it's not needed as we're not using `babel`

Comment (u-hubar) [reply .eslintrc.js:9]: We don't have it on the `master` branch

**Review by NZT48 — APPROVED** [review]

**Review by Mihajlo-Pavlovic — APPROVED** [review]
